src/input_box/html/inputbox.html

```diff
@@ -31,7 +31,7 @@
 </head>
 <body>
 
-  <div id="app" v-on:keyup.enter="accept" v-on:keyup.esc="cancel" class="container-fluid">
+  <div id="app" v-on:submit.prevent="onSubmit" v-on:keydown.enter="accept" v-on:keyup.esc="cancel" class="container-fluid">
```

Comment:
Accept is now `keydown`, but Cancel is still `keyup`. Is there are reason for these event triggering to different phases of the key events?

Reply:
Just re-tested it. The [Enter] for 'accept' definitely needs to be on **keydown**. Otherwise pressing [Enter] on dropdown boxes will drop down the list instead of submitting the dialog. So we need to catch it before it goes to the dropdown control. Escape/cancel might work either way.

Reply:
Can we make ESC also trigger on key down? For a more consistent behaviour?

Reply:
Initial test indicates yes, both enter and escape work fine on keydown and I have committed this as another change to the pull request,